### PR TITLE
Add dutch (nl) locale

### PR DESCRIPTION
**Pull request by samvandamme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ For a quick intro to cron see Quartz [Cron Tutorial](http://www.quartz-scheduler
 * Italian
 * Romanian
 * Russian
+* Dutch
```
Comment (alpinweis) [inline README.md:32]: please put it in alpha sorted order

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+### [0.15.0] - 2024-10-20
```
Comment (alpinweis) [inline CHANGELOG.md:1]: also bump it in version.rb

File: spec/exp_descriptor_nl_spec.rb [hunk 3]
```diff
@@ -0,0 +1,431 @@
+# encoding: utf-8
```
Comment (alpinweis) [inline spec/exp_descriptor_nl_spec.rb:1]: use a non-english version of the spec as the basis. each of them should have 405 lines

**Review by alpinweis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (samvandamme) [reply README.md]: Got it, I thought because Brazilian was below English it was added-date.

File: CHANGELOG.md (hunk 2 above)
Comment (samvandamme) [reply CHANGELOG.md:1]: Done.

File: spec/exp_descriptor_nl_spec.rb (hunk 3 above)
Comment (samvandamme) [reply spec/exp_descriptor_nl_spec.rb:1]: I made the change by removing `strict_quartz` and `timezone` to end up on 405 lines.

**Pull request merged**